src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -2,7 +1,26 @@
+import { Visible } from '../../types/Visible';
+
+type Props = {
+  visible: Visible;
+  setVisible: (arg: Visible) => void;
+  filterBy: string;
+  setFilterBy: (arg: string) => void;
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  visible = 'all',
+  setVisible,
+  filterBy = '',
```

Comment:
Similar to the `visible` prop, assigning a default value directly in the destructuring assignment is not recommended. Handle default values outside of the destructuring.